w3lib/url.py

```diff
@@ -85,13 +85,13 @@ def safe_url_string(url, encoding='utf8', path_encoding='utf8', quote_path=True)
     if quote_path:
         path = quote(to_bytes(parts.path, path_encoding), _path_safe_chars)
     else:
-        path = to_native_str(parts.path)
+        path = to_unicode(parts.path)
```

Comment:
I wonder if some of these calls are redundant now, as Python 2 support is dropped.